app/views/handlers/feedback.py

```diff
@@ -211,32 +225,118 @@ def is_enabled(schema: QuestionnaireSchema) -> bool:
 
 
 class FeedbackMetadata:
-    def __init__(self, feedback_count, form_type, language_code, region_code, tx_id):
-        self.feedback_count = feedback_count
-        self.feedback_submission_date = datetime.now(timezone.utc).strftime("%Y-%m-%d")
-        self.form_type = form_type
-        self.language_code = language_code
-        self.region_code = region_code
+    def __init__(self, case_id: str, tx_id: str):
+        self.case_id = case_id
         self.tx_id = tx_id
 
     def __call__(self) -> Mapping:
```

Comment:
Worth updating this to be explicit:
```suggestion
    def __call__(self) -> dict[str, str]
```